docs/_sass/colors.scss

```diff
@@ -0,0 +1,143 @@
+// This is forked from an internal file that is used to generate Glue.
+// These variables don't appear in the generated Glue css, which is why
+//   this copy is needed.
+// Please don't edit this file since it would create a maintenance
+//   burden for people trying to keep it up to date with Glue.
```

Comment:
"Do not edit this file" is a good message. Would you further add a comment on what should be done in case we want to update this file?

Reply:
It's located on git on borg, so I can't reference the original source publicly. Do you have a preference on how to provide the reference to people who need it?

Reply:
'Search for "glue project" in git-on-borg' or using "go" link would be nice.

Reply:
I will add a go link.